cypress/tests/file-download.cy.ts

```diff
@@ -0,0 +1,27 @@
+import { FileDownloadPage } from '../support/page-objects/file-download.po';
+
+const DOWNLOAD_FILEPATH = Cypress.config().downloadsFolder;
+
+describe('File Download Page Functionality', () => {
+  beforeEach(() => {
+    cy.task('deleteDownloads');
+    FileDownloadPage.visit();
+  });
+
+  it('Should download first available file in list', () => {
+    FileDownloadPage.getAvailableFiles()
+      .first()
+      .click()
+      .invoke('text')
+      .then((fileName) => {
+        cy.assertFileDownload(fileName);
+      });
+  });
+
+  it('Should download all available files successfully', () => {
```

Comment:
This page is inconsistent, and will refresh the file list occasionally. Downloading every single file is also a little overboard. Due to the inconsistent data set and overtesting, I think this one should be skipped. The single download test should be sufficient for testing this page's functionality

Reply:
Done